aixplain/modules/pipeline/asset.py

```diff
@@ -152,41 +165,42 @@ def run(
         timeout: float = 20000.0,
         wait_time: float = 1.0,
         **kwargs,
-    ) -> Dict:
-        """Runs a pipeline call.
-
-        Args:
-            data (Union[Text, Dict]): link to the input data
-            data_asset (Optional[Union[Text, Dict]], optional): Data asset to be processed by the pipeline. Defaults to None.
-            name (Text, optional): ID given to a call. Defaults to "pipeline_process".
-            timeout (float, optional): total polling time. Defaults to 20000.0.
-            wait_time (float, optional): wait time in seconds between polling calls. Defaults to 1.0.
-            kwargs: A dictionary of keyword arguments. The keys are the argument names
-
-        Returns:
-            Dict: parsed output from pipeline
-        """
+    ) -> PipelineResponse:
         start = time.time()
         try:
             response = self.run_async(data, data_asset=data_asset, name=name, **kwargs)
-            if response["status"] == "FAILED":
+            if response["status"] == Status.FAILED: 
```

Comment:
Here, are we sure the value of "status" in the matching enum type or a string only?

Reply:
The enum makes sure that if it's passed in as string it turns it into enum object